content/study/posts/liberia_custom.html

```diff
@@ -0,0 +1,997 @@
+<!DOCTYPE html><html><head><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1"><style>@font-face {
```

Comment:
This file is not used by the application

Reply:
ok, not sure how this ended up here.